targets/TARGET_STM/TARGET_STM32F3/TARGET_STM32F303x8/TARGET_NUCLEO_F303K8/PinNames.h

```diff
@@ -104,6 +104,8 @@ typedef enum {
     LED3        = PB_3,
     LED4        = PB_3,
     USER_BUTTON = 0x20, // no user button on the board
```

Comment:
This pin is specified as not existing, left the new button map in but could also pull it out since it's not really needed.